src/Docusaurus/versioned_docs/version-v3/msbuild/authoring_first_msbuild_project.md

```diff
@@ -13,31 +13,31 @@ If you want to copy and paste this example, remember to change the &lt;ProjectGu
-                &lt;</font><font size="2" color="#A31515">IntermediateOutputPath</font><font size="2" color="#0000FF">&gt;</font><font size="2">obj\$(Configuration)\</font><font size="2" color="#0000FF">&lt;/</font><font size="2" color="#A31515">IntermediateOutputPath</font><font size="2" color="#0000FF">&gt;
-                &lt;</font><font size="2" color="#A31515">DefineConstants</font><font size="2" color="#0000FF">&gt;</font><font size="2">Debug</font><font size="2" color="#0000FF">&lt;/</font><font size="2" color="#A31515">DefineConstants</font><font size="2" color="#0000FF">&gt;
-        &lt;/</font><font size="2" color="#A31515">PropertyGroup</font><font size="2" color="#0000FF">&gt;
-        &lt;</font><font size="2" color="#A31515">PropertyGroup</font><font size="2" color="#0000FF"> </font><font size="2" color="#FF0000">Condition</font><font size="2" color="#0000FF">=</font><font size="2">"</font><font size="2" color="#0000FF"> '$(Configuration)|$(Platform)' == 'Release|x86' </font><font size="2">"</font><font size="2" color="#0000FF">&gt;
-                &lt;</font><font size="2" color="#A31515">OutputPath</font><font size="2" color="#0000FF">&gt;</font><font size="2">bin\$(Configuration)\</font><font size="2" color="#0000FF">&lt;/</font><font size="2" color="#A31515">OutputPath</font><font size="2" color="#0000FF">&gt;
-                &lt;</font><font size="2" color="#A31515">IntermediateOutputPath</font><font size="2" color="#0000FF">&gt;</font><font size="2">obj\$(Configuration)\</font><font size="2" color="#0000FF">&lt;/</font><font size="2" color="#A31515">IntermediateOutputPath</font><font size="2" color="#0000FF">&gt;
-        &lt;/</font><font size="2" color="#A31515">PropertyGroup</font><font size="2" color="#0000FF">&gt;
-        &lt;</font><font size="2" color="#A31515">ItemGroup</font><font size="2" color="#0000FF">&gt;
-                &lt;</font><font size="2" color="#A31515">Compile</font><font size="2" color="#0000FF"> </font><font size="2" color="#FF0000">Include</font><font size="2" color="#0000FF">=</font><font size="2">"</font><font size="2" color="#0000FF">Product.wxs</font><font size="2">"</font><font size="2" color="#0000FF"> /&gt;
-        &lt;/</font><font size="2" color="#A31515">ItemGroup</font><font size="2" color="#0000FF">&gt;
-        &lt;</font><font size="2" color="#A31515">Import</font><font size="2" color="#0000FF"> </font><font size="2" color="#FF0000">Project</font><font size="2" color="#0000FF">=</font><font size="2">"</font><font size="2" color="#0000FF">$(WixTargetsPath)</font><font size="2">"</font><font size="2" color="#0000FF"> /&gt;
-    &lt;/</font><font size="2" color="#A31515">Project</font><font size="2" color="#0000FF">&gt;</font>
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+        <PropertyGroup>
```

Comment:
You'll want to fix the indentation. The fence (```) will treat all whitespace as important so need to left justify everything now.

Reply:
Indentation is correct, expressing the tag hierarchy. In fact, it reflects exactly the copy/paste from the V4 documentation

Why left justify? Unless you meant changing tabs to spaces...

Reply:
The open and close of the `Project` element do not align. And yes, ideally, everything is two spaces (not tabs) to best fit in the documentation window.